pkg/releaser/releaser_test.go

```diff
@@ -466,6 +472,7 @@ func TestReleaser_CreateReleases(t *testing.T) {
 				assert.Equal(t, assetPath, fakeGitHub.release.Assets[0].Path)
 				assert.Equal(t, tt.commit, fakeGitHub.release.Commit)
 				assert.Equal(t, tt.latest, fakeGitHub.release.MakeLatest)
+				assert.Equal(t, tt.prerelease, fakeGitHub.release.Prerelease)
```

Comment:
can we have a test for the pre-release true?

Reply:
agree